demos/android-supabase-todolist/app/src/main/java/com/powersync/androidexample/App.kt

```diff
@@ -26,25 +31,57 @@ import com.powersync.demos.screens.SignUpScreen
 import com.powersync.demos.screens.TodosScreen
 import kotlinx.coroutines.runBlocking
 
-
 @Composable
-fun App() {
+fun App(
+    cameraService: CameraService,
+    attachmentDirectory: String,
+) {
     val driverFactory = rememberDatabaseDriverFactory()
-    val supabase = remember {
-        SupabaseConnector(
-            powerSyncEndpoint = BuildConfig.POWERSYNC_URL,
-            supabaseUrl = BuildConfig.SUPABASE_URL,
-            supabaseKey = BuildConfig.SUPABASE_ANON_KEY
-        )
-    }
-    val db = remember { PowerSyncDatabase(driverFactory, schema) }
+    val supabase =
+        remember {
+            SupabaseConnector(
+                powerSyncEndpoint = BuildConfig.POWERSYNC_URL,
+                supabaseUrl = BuildConfig.SUPABASE_URL,
+                supabaseKey = BuildConfig.SUPABASE_ANON_KEY,
+                storageBucket = BuildConfig.SUPABASE_ATTACHMENT_BUCKET,
+            )
+        }
+
+    val db = remember { PowerSyncDatabase(driverFactory, schema, dbFilename = "333.sqlite") }
```

Comment:
Is the `dbFilename` a leftover from debugging?

Reply:
Jip. Good catch. Removed this.